src/plumbing/checkId.ts

```diff
@@ -7,18 +7,32 @@ const prefixes = new Map<ResourceKind, string>([
   ['mandate', 'mdt_'],
   ['order', 'ord_'],
   ['orderline', 'odl_'],
+  ['organization', 'org_'],
   ['payment', 'tr_'],
+  ['profile', 'pfl_'],
   ['refund', 're_'],
   ['shipment', 'shp_'],
   ['subscription', 'sub_'],
 ]);
+/**
+ * Retruns whether the passed permission identifier seems plausible (`true`); or is definitely invalid (`false`).
+ */
+const checkPermissionId = (() => {
+  const pattern = /^[a-z.]+$/;
+  return function checkPermissionId(value: string) {
+    return pattern.test(value);
+  };
+})();
 /**
  * Returns whether the passed identifier seems plausible (`true`); or is definitely invalid (`false`).
  */
 export default function checkId(value: string | undefined, resource: ResourceKind): value is string {
   if (value == undefined) {
     return false;
   }
+  if (resource == 'permission') {
```

Comment:
Hmm, not sure if this is really necessary. Let's let the API handle this.

Reply:
Yes, I agree that this might be a little excessive. I'll take this out.